kuksa_databroker/databroker/src/main.rs

```diff
@@ -247,14 +247,15 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
                 .value_name("PORT")
                 .required(false)
                 .env("KUKSA_DATA_BROKER_PORT")
-                .value_parser(clap::value_parser!(u16).range(1..))
+                .value_parser(clap::value_parser!(u16))
```

Comment:
my understanding is that the port number `0` can be used to bind to *any free port*? If so, then I think this is worth mentioning in the property's description both in the source code as well as in the README

Reply:
Yes, that is correct (and the reason I removed the limit of > 0).

With that said, it doesn't really do much good at the moment as databroker isn't printing which port it binds to (when using port 0) as it's just printing the port provided verbatim..

I have a plan to fix it, but as it's a bit involved I didn't do it as part of this PR.
Regarding documenting this behaviour, I mean sure it doesn't hurt, but it's standard unix behaviour, right?